android/app/build.gradle

```diff
@@ -91,6 +91,7 @@ dependencies {
     compile 'com.jakewharton:butterknife:8.4.0'
     compile 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
     apt 'com.jakewharton:butterknife-compiler:8.4.0'
+    compile 'net.opacapp:multiline-collapsingtoolbar:1.3.1'
```

Comment:
The another one is by creating a custom collapsing toolbar similar to one we are using in speaker's activity title! and I tried and it look similar to this but for that, we need to change the menu items `app:showAsAction="ifRoom"` to`never`. Shall I update the PR with this solution?